post_covid_stats.py

```diff
@@ -6,23 +6,29 @@
 from slack_bot import CovidSlackBot
 
 SELECTED_CODES = os.environ.get("SELECTED_CODES", 'VIC,NSW')
+POPULATION_BRACKET = os.environ.get("POPULATION_BRACKET", "16+")
 SLACK_CHANNEL_NAME = os.environ.get("SLACK_CHANNEL_NAME", "#testcovidbot")
 SLACK_BOT_TOKEN = os.environ.get("SLACK_BOT_TOKEN", '')
 SLACK_BOT_EMOJI = os.environ.get("SLACK_BOT_EMOJI", ":robot_face:")
 SLACK_BOT_NAME = os.environ.get("SLACK_BOT_NAME", "CovidLiveSummary")
 
 
 def post_covid_stats() -> SlackResponse:
+    population_data_file: str = "resources/population.json"
     source_url: str = "https://covidlive.com.au/covid-live.json"
     covid_data: Dict = fetch_and_parse_data(source_url)
-
+    
+    with open(population_data_file) as populationDataFile:
+        population_data: Dict = json.load(populationDataFile);
+        populationDataFile.close();
```

Comment:
Shouldn't need to explicitly close when using a with block IIRC

Reply:
My python foo is lacking, as you can see.